code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -288,3 +288,19 @@
 	new /obj/item/weapon/storage/box/lethalshot(src)
 	new /obj/item/weapon/storage/box/lethalshot(src)
 	new /obj/item/weapon/storage/box/lethalshot(src)
+	
+/obj/structure/closet/secure_closet/lowsec
+	name = "emergency security supplies"
+	req_access = list(access_security)
+	icon_state = "tac"
+	
+/obj/structure/closet/secure_closet/lowsec/New()
+	..()
+	new /obj/item/weapon/implant/adrenalin(src)

```

Comment:
This is a item with syndie tech levels, not a great idea to have starting on the station.